browser/css/jsdialogs.css

```diff
@@ -568,6 +573,25 @@ td.jsdialog > [id^='table-box']:not(.sidebar) {
 	/* grid-column: '1 / ' + (this._columns + 1) - set inside JS */
 }
 
+#ServerAuditDialog .ui-treeview-headers {
+	display: grid;
+	grid-template-columns: auto 1fr 170px;
+}
+
+#ServerAuditDialog .ui-treeview-icon {
+	margin-inline-end: 2px;
+}
```

Comment:
We don't need margins if we use column-gap as in:

```css
.ui-treeview-headers,
+.ui-treeview-entry {
+	column-gap: 8px;
+}
```

Reply:
removed margin-inline-end